pipelines/update_geojson.py

```diff
@@ -0,0 +1,59 @@
+import geopandas as gpd
+import pandas as pd
+
+from src.constants import ISO3S
+from src.datasources import codab
+from src.utils import blob
+
+ADMS = [0, 1, 2]
+
+
+def clean_gdf(gdf):
+    gdf["name"] = gdf.apply(
+        lambda row: (
+            row[f"ADM{adm}_EN"]
+            if pd.notna(row.get(f"ADM{adm}_EN"))
+            and row.get(f"ADM{adm}_EN") != ""
+            else row.get(f"ADM{adm}_FR", "")
```

Comment:
Tiny thing but could we change this to:
```
row[f"ADM{adm}_FR"]
if pd.notna(row.get(f"ADM{adm}_FR"))
and row.get(f"ADM{adm}_FR") != ""
else row.get(f"ADM{adm}_EN", "")
``` 
just to prioritize the French where we have it? Apologies I should've written it this way around in the first place!